libminifi/include/utils/MinifiConcurrentQueue.h

```diff
@@ -135,6 +143,22 @@ class ConcurrentQueue {
   std::deque<T> queue_;
 };
 
+// Enables batched operations on a ConcurrentQueue by holding a reference to the queue and locking it's mutex
```

Comment:
Very minor, but you probably meant "its" instead of it's.

Reply:
Good point.